test/lib/pgBackRestTest/Common/ContainerTest.pm

```diff
@@ -482,7 +482,7 @@ sub containerBuild
             {
                 $strScript .=
                     "    echo 'deb http://apt.postgresql.org/pub/repos/apt/ " .
-                    $$oVm{$strOS}{&VM_OS_REPO} . '-pgdg main' .
+                    $$oVm{$strOS}{&VM_OS_REPO} . '-pgdg main' . ($strOS eq VM_U18 ? ' 14' : '') .
```

Comment:
Is this something that will have to change every time now for u18 when a new PG version is added? We don't have pg version 14 yet so I'm not 100% clear on what this is doing.

Reply:
This adds the pre-release repo for PG14. Once it is prod it will move into main and we can remove the extra `14` repo. This happens each year (see history for details).

This would be better as a setting in VmTest.pm, but so it is for now.

Reply:
ok